pkg/aqua/command.go

```diff
@@ -85,6 +85,15 @@ func (c *command) Scan(imageRef ImageRef) (report ScanReport, err error) {
 		fmt.Sprintf("--jsonfile=%s", reportFile.Name()),
 	}
 
+	switch c.cfg.ScannerCLIRegisterImages {
```

Comment:
What if it's set to an invalid choice?

Reply:
Interestingly it's accepted without any error or warning as depicted below:

![validate_env](https://user-images.githubusercontent.com/1322923/107488406-9933d500-6b87-11eb-94d6-da44b34e06a8.png)

To fail fast I've changed this behavior and added custom parse function so the service won't start with an invalid config value:

```
{"built_at":"unknown","commit":"none","level":"info","msg":"Starting harbor-scanner-aqua","time":"2021-02-10T09:04:25Z","version":"dev"}
{"level":"fatal","msg":"Error: getting config: env: parse error on field \"ScannerCLIRegisterImages\" of type \"etc.ImageRegistration\": expected values Never, Always or Compliant but got XXX","time":"2021-02-10T09:04:25Z"}
```